src/r2mm/manager/ManagerSettings.ts

```diff
@@ -27,6 +28,17 @@ export default class ManagerSettings {
         return this.LOADED_SETTINGS;
     }
 
+    /**
+     * Return default game selection needed to skip the game selection screen.
+     */
+    public static async getDefaults(settings?: ManagerSettings) {
```

Comment:
Is it ok to import GameManager in ManagerSettings, or should the files be kept loosely coupled? I personally think the helper methods fits nicely here, but it could be moved to another file that imports both GameManager and ManagerSettings, I guess?

Reply:
I'd say no mostly because then we have a two-way binding on the files. It'd be nice if we could keep settings pretty independent of everything.

What does this do when there's no defaultGame?

Reply:
I'll see if there's a suitable helper file to move this into, and create one if there isn't.

If there's no default game, the returned `defaultGame` and `defaultPlatform` will be undefined (the latter can be undefined independently too). If either is undefined, the calling Vue component will not set `this.selectedGame`/`this.selectedPlatform` nor call `this.proceed` (i.e. works like it did before, I believe).

Reply:
Created [ManagerUtils ](https://github.com/ebkr/r2modmanPlus/pull/849/files#diff-5f7b44feef6c321d50703c5cd6d8cce0441680e1cdfde8d25f96082187bccd51)